docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: "3"
+services:
+
+    redaxo:
+        image: friendsofredaxo/redaxo:5
+        ports:
+            - 80:80
```

Comment:
Ich wollte es spaßeshalber gerade mal testen. Ging aber nicht, da Port 80 bei mir schon belegt.
Sollte man da einen anderen nehmen, oder ist das so gut und sinnvoll, da wer mit Docker arbeitet, Port 80 in der Regel frei hat?

Reply:
Ich habe Port 80 immer belegt

Reply:
Ich bin mir unsicher, was hier besser ist. Ich selbst würde Port 80 auch nicht unbedingt benutzen, aber er hat den Vorteil, dass man ihn nicht explizit angeben muss, sondern dass `localhost` alleine ausreicht.

Letztendlich war meine Motivation, diese Docker-Files hinzuzufügen, vor allem die, dass Docker-affine Entwickelnde sehen, dass es REDAXO-Images im Hub gibt, und dass sie damit arbeiten können.

In der Praxis werden die Leute sowas wie den Port vermutlich für sich anpassen? Wie seht ihr das?

Reply:
Meine Motivation wäre wenn schon, dass man es konkret nutzen kann, um an REDAXO zu entwickeln.
Also dass die, die Docker nutzen, und keinen lokalen Server haben etc., trotzdem schnell dieses Repo lauffähig bekommen.

> In der Praxis werden die Leute sowas wie den Port vermutlich für sich anpassen? Wie seht ihr das?

Das fände ich als Nutzer recht nervig, wenn ich da dann die ganze Zeit eine geänderte Datei rumliegen hätte, die ich aber nicht committen darf.

Habe gerade mal probiert, wie wäre es mit sowas?

```yml
        ports:
            - ${REDAXO_PORT:-80}:80
```

So ist es default Port 80. Aber man kann es so ändern:
`REDAXO_PORT=8080 docker-compose up -d`
(oder irgendwo global die env var setzen)

Reply:
Sehr gute Idee, @gharlan!
Dann sollten wir vielleicht noch eine kurze Info dazu in der README hinterlegen.